docker-compose.yml

```diff
@@ -219,7 +219,7 @@ services:
 
   # Metabase: Reporting paltform
   metabase:
-    image: metabase/metabase:v0.31.2
+    image: zouchine/metabase:arm-0.31.2
```

Comment:
We'll want to provide the Dockerfile locally in the project instead of using a Docker Hub repo.